package.json

```diff
@@ -73,5 +78,28 @@
     "node": ">= 6.9.1",
     "npm": ">= 3.10.3",
     "yarn": ">= 0.20.1"
+  },
+  "jest": {
+    "automock": true,
+    "verbose": true,
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "app/javascript/packs/**/*.js"
+    ],
+    "coverageReporters": [
+      "lcov"
+    ],
+    "moduleNameMapper": {
+      "^.+\\.(png|gif|css|scss)$": "identity-obj-proxy"
```

Comment:
I'd usually mock binary assets, like images & fonts, like this:

```
"\.(png|jpg|gif|svg|eot|ttf|woff|woff2)$": "jest.mock.static-file.js"
```
where the file mentioned above would contain

```
'use strict';
module.exports = 'static-file-stub';
```

so that when someone `import`s a binary asset in JS code, he gets back the `static-file-stub` string.

TL;DR not sure if `identity-obj-proxy` is worth using on binary assets.